.github/workflows/monthly.yml

```diff
@@ -0,0 +1,41 @@
+name: monthly
+
+on:
+  push
+    branches:
+      - master
+
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  run:
+    name: Refresh month of man pages
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler: none
+
+      - name: Install dependencies
+        run: rake setup
```

Comment:
Use `working_directory: ./bundler` so that this runs on the proper folder. Also, I think you want `bin/rake spec:deps` here? Or actually maybe `gem install ronn:'~> 0.7.3'` is enough?

Reply:
Okay, I can check this in a fresh container, I'll change it to whatever works! :smile: